### PR TITLE
New package: mdBook-0.0.21

### PR DESCRIPTION
**Pull request by hwangh95**

### CODE REVIEW

**Comment by cr6git** [comment]
`hostmakedepends="cargo rust"`

`rust` is automatically pulled by `cargo`.

**Comment by the-maldridge** [comment]
I'm suspicious of nocross=yes.  If this is something systemic with rust I would rather set it on the build style.

**Comment by hwangh95** [comment]
I don't know if it is systematic, I saw that other rust packages had it so I wrote it that way. Thought it might break without it.

**Comment by the-maldridge** [comment]
Please remove it and force push, if it turns out it needs it, you'll be able to see things break on Travis.  Since this just processes files though I don't think it should be architecture dependent.

File: srcpkgs/mdBook/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'ripgrep'
```
Comment (the-maldridge) [inline srcpkgs/mdBook/template]: This doesn't look like ripgrep to me.

File: srcpkgs/mdBook/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+pkgname=mdBook
+version=0.0.21
+revision=1
+hostmakedepends="cargo rust"
```
Comment (the-maldridge) [inline srcpkgs/mdBook/template]: rust is a dependency of cargo, please remove it from this list.

File: srcpkgs/mdBook/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+short_desc="Create book from markdown files. Like Gitbook but implemented in Rust"
+maintainer="Huidong Hwang <hwangh95@gmail.com>"
+license="MPL-2.0"
+homepage="https://github.com/rust-lang-nursery/${pkgname}"
+distfiles="https://github.com/rust-lang-nursery/${pkgname}/archive/${version}.tar.gz"
+checksum=bffb39106ab9f546c7a02667aa3616c06f05dfb566ef4830f51fd244c7986239
+#nocross=yes
```
Comment (the-maldridge) [inline srcpkgs/mdBook/template]: This should indeed be nocross.  Perhaps provide a brief comment that this cannot be cross compiled due to limitations in cargo at this time.

File: srcpkgs/mdBook/template [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+do_build() {
+	cargo build --release
+}
+do_install() {
```
Comment (the-maldridge) [inline srcpkgs/mdBook/template]: newline between the end of the previous function and the start of this one.

File: srcpkgs/mdBook/template (hunk 1 above)
Comment (hwangh95) [reply srcpkgs/mdBook/template]: Yeah it doesn't lol

**Pull request closed without merging**